schemas/trademark-registration-schema.json

```diff
@@ -0,0 +1,200 @@
+  "properties": {
+    "register": {
+      "description": "The register on which the trademark is listed",
+      "type": "string"
+    },
+    "holders": {
+      "type": "array",
+      "description": "Details of the holder(s) of the trademark (sometimes called the owners)",
+      "items": {
+        "$ref": "includes/entity.json"
+      }
+    },
+    "correspondent": {
+      "$ref": "includes/entity.json",
+      "description": "Details of the correspondent (i.e. contact person)"
+    },
+    "representative": {
+      "$ref": "includes/entity.json",
+      "description": "Details of the representative"
+    },
+    "mark_details": {
+      "description": "Details of the trademark itself (rather than the registration)",
+      "type": "object",
+      "properties": {
+        "mark_text": {
+          "description": "Text of the trademark",
+          "type": "string"
+        },
+        "mark_form": {
+          "description": "The form(s) (i.e. category) of the mark. Sensory marks include smells and sounds",
+          "type": "string",
+          "enum": [
+            "text",
+            "image",
+            "sensory_mark",
+            "three_d_mark"
+          ]
+        },
+        "mark_type": {
+          "description": "Free text description of mark type (as provided by register), e.g. 'Illustration', 'Typeset: Word(s)/letter(s)/number(s)'",
+          "type": "string"
+        },
+        "mark_image": {
+          "properties": {
+            "uid": {
+              "description": "A unique identifier",
+              "type": "string"
+            },
+            "format": {
+              "description": "Type of image format",
+              "type": "string"
+            },
+            "path": {
+              "description": "Path of image",
+              "type": "string"
+            },
+            "description": {
+              "description": "Description of image in words",
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        },
+        "classification": {
+          "type": "array",
+          "description": "Classification of the trademark",
+          "items": {
+            "$ref": "#/definitions/nice_classification"
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "status": {
+      "description": "Status of the trademark",
+      "type": "string"
+    },
+    "uid": {
+      "description": "Unique id/reference for the trademark registration",
+      "type": "string"
+    },
+    "url": {
+      "description": "URL of trademark",
```

Comment:
(Same above (in "status") too. Maybe just check for all mentions of "trademark".)